device-types/Intel/H2312XXLR2.yaml

```diff
@@ -0,0 +1,32 @@
+---
+manufacturer: Intel
+model: H2312XXLR2
+slug: intel-h2312xxlr2
+u_height: 2
+is_full_depth: true
+comments: |
+  [H2312XXLR2 Specifications on intel.com](https://www.intel.com/content/www/us/en/products/sku/87573/intel-server-chassis-h2312xxlr2/specifications.html)
+subdevice_role: parent
+device-bays:
+  - name: Node 1
+    label: Blade 1
+  - name: Node 2
+    label: Blade 2
+  - name: Node 3
+    label: Blade 3
+  - name: Node 4
+    label: Blade 4
+power-ports:
```

Comment:
It looks like the PSUs are actually modular, so these should be removed from this file and created under `module-bays`. Then you should add the PSUs themselves to the module-types folder

According to the datasheet, the PSUs are `FXX2130PCRPS`